### DISCUSSION TITLE
Open Questions and FAQ 🙋🏻‍♀️🤔

### DISCUSSION THREAD

**Discussion by NS007-dev** [App-Choreography/Support#2] [Q&A]
If you have any general questions, that you the community can help solve, share below 👇

**Comment by hrshbrdhnn** [comment]
I'm learning to Android development using kotlin. And I am facing some issues with recyclerview items. Can someone please help me with that?

**Reply by NS007-dev** [reply]
@harsh1x4 so I haven't got experience with Android Development or Kotlin, but I will do research about recyclerview items and share the sources. 
If anyone else may have the answer to this question feel free to answer and share idea below.
Thank you,

**Reply by NS007-dev** [reply]
If you have any specific problems with recyclerview items in android development please can you point them out so I can do specific research on it @harsh1x4 ? Thanks

**Reply by NS007-dev** [reply]
some resources I found in general about recyclerview items in android development:

https://developer.android.com/develop/ui/views/layout/recyclerview
https://guides.codepath.com/android/using-the-recyclerview
https://medium.com/swlh/common-mistake-developers-make-when-building-recyclerviews-and-how-to-fix-it-using-3b642a2ba0f4

**Reply by hrshbrdhnn** [reply]
Thank you, @CodingSpecies , for these amazing resources. And I have got the solution to what I was looking for. Thanks again.

**Comment by Intelligence08** [comment]
Hi! I'd be happy to help you out with your RecyclerView issue in Kotlin.

**Comment by Intelligence08** [comment]
Could you please provide a bit more detail about the problem you're facing? For example:

A brief description of what you're trying to achieve
Looking forward to your response!
